parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -2111,8 +2111,8 @@ parseSearchType input typ = fmap Type.removeAllEffectVars <$> parseType input ty
 parseType :: (Monad m, Var v)
   => Input -> String -> Action' m v (Either (Output v) (Type v Ann))
 parseType input src = do
-  -- `show Input` is the name of the "file" being lexed
-  (names0, lexed) <- lexedSource (Text.pack $ show input) (Text.pack src)
+  -- `inputName input` is the name of the "file" being lexed
+  (names0, lexed) <- lexedSource (inputName input) (Text.pack src)
```

Comment:
Is this nicer?  Why not stick with `show input` here?

Reply:
Good call, since it's just a stub impl it seems fine to `show input` still (showing uglier names)